R/build-reference.R

```diff
@@ -174,7 +174,7 @@ build_reference <- function(pkg = ".",
     old_dir <- setwd(path(pkg$dst_path, "reference"))
     on.exit(setwd(old_dir), add = TRUE)
 
-    old_opt <- options(width = 80)
+    old_opt <- options(width = 10000)
```

Comment:
Can we leave the width as is? I think this better captures what people see in a console.

Reply:
Actually leaving it at 80 will automatically wrap long lines which means that the css will not take effect.

Reply:
That’s not entirely true as many functions don’t respect width. 

I don’t think we want to simulate and incredibly wide console.

Reply:
the default `print` function for data.frames respects `width` which leads to sometimes ugly output in the code block because the data.frame is split. keeping width at 80 renders these changes obsolete.

Reply:
I understand that, but I don't think it's a good idea to force people to scroll vertically.

Reply:
Maybe exposing it as an argument would be the best way, as this decision must be left to the package developer as to how the output shows in the docs. If said flag is set to `TRUE` than the width would be set to max.

Reply:
I'm sorry but I don't agree — I think it's better for the output to match what you're likely to see in a console, which is not going to be 10,000 characters wide.